modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -145,76 +145,6 @@ users filter ((item, index) -> item.error == null)
 ]
 ----
 
```

Comment:
This removed example probably should be restored. I was not in the source but probably should've been.

Reply:
A workaround is to restore it to this page.